app/models/annotation.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Annotation < ApplicationRecord
+  SUPPORTED_ANNOTATIONS = %w[managed].freeze
+
+  belongs_to :annotated, polymorphic: true
```

Comment:
```suggestion
  belongs_to :annotated, polymorphic: true, required: true, inverse_of: :annotations
```

Reply:
actually `required: true` is the default but I think it is alright to have it set for readability. As you wish.

Reply:
I'll commit this